flake.nix

```diff
@@ -33,9 +33,19 @@
       url = "github:Mic92/nixos-shell";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+    naersk = {
+      url = "github:nix-community/naersk";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    fenix = {
+      url = "github:nix-community/fenix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
     deadnix = {
       url = "github:astro/deadnix";
       inputs.nixpkgs.follows = "nixpkgs";
+      inputs.naersk.follows = "naersk";
```

Comment:
building rust. I deduplicated nixpkgs flakes... because astro removed pinnings: https://github.com/astro/deadnix/commit/0531644416d3fa90552e4b85ae5ce684bd6c05d2